src/SourceBuild/tarball/content/repos/Directory.Build.targets

```diff
@@ -564,6 +564,56 @@
     <WriteLinesToFile File="$(RepoCompletedSemaphorePath)RemoveBuiltPackagesFromCache.complete" Overwrite="true" />
   </Target>
 
+  <Target Name="DirSizeBeforeBuild"
```

Comment:
Consider prefixing the targets with a verb such as `Display`

Reply:
Fixed.